ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1532,8 +1532,7 @@ const RuntimePanel = fp.flow(
     const pdSize = pdExists ? persistentDisk.size : defaultDiskSize;
     const initialAutopauseThreshold =
       existingRuntime.autopauseThreshold || DEFAULT_AUTOPAUSE_THRESHOLD_MINUTES;
-    const gpuConfig =
-      gceConfig && gceConfig.gpuConfig ? gceConfig.gpuConfig : null;
+    const gpuConfig = gceConfig?.gpuConfig ? gceConfig.gpuConfig : null;
```

Comment:
can maybe do a little better here?

Reply:
nah, this is used in a few places and I'm not sure of the implications of undefined vs. null here